### PR TITLE
Added Polish translation.

### PR DESCRIPTION
**Pull request by czater**
Hi @reustle!
I added a Polish translation, please take a look 😄

### CODE REVIEW

File: src/i18n/index.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import { enUS, ja, es, de, ptBR, fr, id } from "date-fns/locale";
```
Comment (liquidx) [inline src/i18n/index.js:1]: I think this is missing the pl import here.

**Comment by liquidx** [comment]
Thank you for the PR. I tried to get this to work but it looks like there's a missing import in the PR. 

If you can update the PR and test it with the "deploy preview" link in the "Checks" list to verify, I can merge this in.

File: src/i18n/index.js (hunk 1 above)
Comment (czater) [reply src/i18n/index.js]: That's right! I fixed it.

**Comment by czater** [comment]
I just updated it! Now it works fine 💯

**Comment by liquidx** [comment]
Thank you so much!

**Pull request merged**